### ISSUE TITLE
Problems with MASTER under Windows

### ISSUE THREAD

**Issue by snaphat** [jszklarz-haventec/makeitdark#36]
Current version of the script doesn't work unless I modify the script as follows (Windows 10) to encapsulate the path for unpacking the asar archive:

`unpack_command = apply_sudo(sudo, "npx asar extract {0}/app.asar {0}/app.asar.unpacked".format('"' + packed_path + '"'))`

and for packing the asar archive after modifications:

`pack_command = apply_sudo(sudo, "npx asar pack {0}/app.asar.unpacked {0}/app.asar".format('"' + packed_path + '"'))`

Also, in the current version of the script I have to create a symbolic link for
`C:\Program Files\Slack`
to the following 
`C:\Program Files\Slack\App-Slack` 

Basically, the detection requires a subdirectory underneath of the Slack directory with the prefix of "App-", but that's not the location it installs to in program files.

It should just be checking "C:\Program Files\Slack" directly for resources.

**Comment by LostConnection** [comment]
Mmm I see, I'll have another look at a fresh install on Windows 10.
